### PR TITLE
OCPBUGS-25362: kubelet: fix kubelet labels

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

File: templates/common/_base/units/kubelet-selinux-relabel.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  Description=Relabel necessary files for kubelet
+  Before=kubelet-dependencies.target
+
+  [Service]
+  Type=oneshot
+  RemainAfterExit=yes
+  User=root
+  ExecStart=/usr/sbin/restorecon /usr/local/bin/kubenswrapper
+  ExecStart=/usr/sbin/restorecon /usr/bin/kubensenter
+  TimeoutSec=300
```
Comment (mrunalp) [inline templates/common/_base/units/kubelet-selinux-relabel.yaml:14]: This could be lower?

**Comment by rphillips** [comment]
Looks like there is a flake with test unit.

/test unit

**Review by mrunalp — APPROVED** [review]
/lgtm

**Pull request merged**